src/types/api.ts

```diff
@@ -141,11 +137,14 @@ export type RectangleAnnotation = AnnotationBase<'rectangle'> & {
   style: AnnotationStyle
 }
 
-export type TextAnnotation = AnnotationBase<'text'> & {
```

Comment:
modeling annotations by geometry and removed `TextAnnotation`. It is more straight forward to just allow shapes to be filled with text via a `content` prop if desired.